contracts/amm/WethHelper.sol

```diff
@@ -0,0 +1,111 @@
+pragma solidity 0.6.12;
+
+import "@openzeppelin/contracts-ethereum-package/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts-ethereum-package/contracts/token/ERC20/SafeERC20.sol";
+
+import "../market/IMarket.sol";
+
+interface IWETH {
+    function deposit() external payable;
+    function transfer(address to, uint value) external returns (bool);
+    function withdraw(uint) external;
+}
+
+interface IMinterAmm {
+    function bTokenBuy(
+        uint256 marketIndex,
+        uint256 bTokenAmount,
+        uint256 collateralMaximum
+    ) external returns (uint256);
+
+    function bTokenSell(
+        uint256 marketIndex,
+        uint256 bTokenAmount,
+        uint256 collateralMinimum
+    ) external returns (uint256);
+
+    function provideCapital(uint256 collateralAmount, uint256 lpTokenMinimum) external;
+
+    function getMarket(uint256 marketIndex) external view returns (IMarket);
+
+    function lpToken() external view returns (ISimpleToken);
+}
+
+contract WethHelper {
+    using SafeERC20 for IERC20;
+
+    address internal constant WETH = 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
+
+    constructor() public {
+
```

Comment:
We should put a `WethHelperCreated(address: helperAddress)` event in here and update the subgraph yaml so that the webapp doesn't need to hardcode the WethHelper addresses, and instead can just fetch it via the subgraph. This means the webapp will be simpler because it doesn't need to differentiate between rinkeby, local, and mainnet.